app/controllers/challenges_controller.rb

```diff
@@ -19,7 +19,8 @@ def index
 
   # GET
   def entries
-    @challengeEntries = ChallengeEntry.includes(:submission).where(challenge_id: @challenge.id).order('created_at DESC').paginate(page: params[:page], per_page: 25).includes(submission: :comments)
+    #@challengeEntries = ChallengeEntry.includes(:submission).where(challenge_id: @challenge.id).order('created_at DESC').paginate(page: params[:page], per_page: 25).includes(submission: :comments)
```

Comment:
Why is this commented out? Just remove it. I'm glad you're using `includes` over the redundant variable, though.